internal/statsd.go

```diff
@@ -15,3 +21,90 @@ type StatsdClient interface {
 	Flush() error
 	Close() error
 }
+
+type Stat struct {
+	Name  string
+	Kind  string
+	Value interface{} //Really, it can only be a "number" type, but wasn't sure if we want to enforce that here yet
```

Comment:
It looks like it'll be an int in every case but one ([WaitDuration](https://pkg.go.dev/database/sql#DBStats)), so interface is probably better here.